includes/Core/Authentication/Authentication.php

```diff
@@ -947,6 +947,10 @@ private function get_rest_routes() {
 		};
 
 		$can_authenticate = function() {
+			return current_user_can( Permissions::AUTHENTICATE ) || current_user_can( Permissions::VIEW_SHARED_DASHBOARD );
+		};
```

Comment:
Should we rename this variable like we did elsewhere? Technically can a shared dashboard viewer authenticate? It's more that they can use the dashboard, but if they're a non-admin they can't actually authenticate, right?

Reply:
I wanted to rename it but not sure what to name this one. The actual route is called authenticate. And the AC has this route. Can you suggest anything here?

Reply:
Ah, fair enough. Let's go verbose with something like `$can_authenticate_or_view_dashboard` if you're cool with that? I just figure `current_user_can( Permissions::AUTHENTICATE )` alone would be `$can_authenticate`, and I wouldn't want other code reading this variable later and thinking it _only_ deals with whether the user can authenticate or not.

Does that sounds okay? 😅

Reply:
@aaemnnosttv suggested against using actual capability name as variable. I have changed it to `can_access_authentication`. Does it sound good to you?

Reply:
@kuasha420 SGTM